[Bug] Calling lcd_clrsrc() from keyboard.c raises error

Trying to interface my LCD with QMK, im following the instructions provided in https://github.com/qmk/qmk_firmware/blob/master/docs/feature_hd44780.md

with HD44780_ENABLE = yes in the rules.mk raises following issue:
The following lines in my keyboard.c:

void matrix_init_kb(void) {
    // put your keyboard start-up code here
    // runs once when the firmware starts up

    matrix_init_user();
    lcd_clrsrc();
//    lcd_init(LCD_DISP_ON_CURSOR_BLINK);
}

cause the error:
-----------------------------------------------------
qmk_firmware$ make kb:default
QMK Firmware 0.8.153
Making kb with keymap default

avr-gcc (GCC) 5.4.0
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Size before:
   text	   data	    bss	    dec	    hex	filename
      0	  22382	      0	  22382	   576e	.build/kb_default.hex

Compiling: keyboards/kb/kb.c                                                                       keyboards/kb/kb.c: In function ‘matrix_init_kb’:
keyboards/kb/kb.c:10:5: error: implicit declaration of function ‘lcd_clrsrc’ [-Werror=implicit-function-declaration]
     lcd_clrsrc();
     ^
cc1: all warnings being treated as errors
 [ERRORS]
 | 
 | 
 | 
tmk_core/rules.mk:383: recipe for target '.build/obj_kb_default/keyboards/kb/kb.o' failed
make[1]: *** [.build/obj_kb_default/keyboards/kb/kb.o] Error 1
Make finished with errors
Makefile:560: recipe for target 'kb:default' failed
make: *** [kb:default] Error 1

--- a/docs/feature_hd44780.md
+++ b/docs/feature_hd44780.md
@@ -50,7 +50,7 @@ LCD_DISP_ON_CURSOR_BLINK : display on, cursor on flashing
 ````
 This is best done in your keyboards `matrix_init_kb` or your keymaps `matrix_init_user`.  
 It is advised to clear the display before use.  
-To do so call `lcd_clrsrc()`.
+To do so call `lcd_clrscr()`.
 
 To now print something to your Display you first call `lcd_gotoxy(column, line)`. To go to the start of the first line you would call `lcd_gotoxy(0, 0)` and then print a string with `lcd_puts("example string")`.
 

--- a/docs/ja/feature_hd44780.md
+++ b/docs/ja/feature_hd44780.md
@@ -55,7 +55,7 @@ LCD_DISP_ON_CURSOR_BLINK : ディスプレイオン、点滅カーソル
 ````
 これはキーボードの `matrix_init_kb` またはキーマップの `matrix_init_user` で行うのが最適です。
 使用前にディスプレイをクリアすることをお勧めします。
-そのためには、`lcd_clrsrc()` を呼びます。
+そのためには、`lcd_clrscr()` を呼びます。
 
 ディスプレイに何かを表示するには、最初に `lcd_gotoxy(column, line)` を呼びます。最初の行の先頭に移動するには、`lcd_gotoxy(0, 0)` を呼び出し、その後 `lcd_puts("example string")` を使って文字列を出力します。
 

